recipes/libhomfly/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libhomfly" %}
+{% set version = "1.02r6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/miguelmarco/libhomfly/releases/download/{{ version }}/libhomfly-{{ version }}.tar.gz
+  sha256: f990d991d29a590c54b09eff68bc209c4cf2b8b12d945351688d2db8a0d30979
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - bdw-gc
+  run:
+    - bdw-gc
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libhomfly${SHLIB_EXT}
+    - test -f $PREFIX/include/homfly.h
+
+about:
+  home: https://github.com/miguelmarco/libhomfly
+  license: Unlicense
```

Comment:
Unlicensed but the family is MIT and it has a license file? I'm confused :smile: